extensions/validation/src/test/java/org/mule/extension/validation/BasicValidationTestCase.java

```diff
@@ -14,6 +14,7 @@
 import static org.junit.Assert.fail;
 import static org.mule.extension.validation.internal.ImmutableValidationResult.error;
 import static org.mule.extension.validation.internal.ValidationExtension.DEFAULT_LOCALE;
+
```

Comment:
intellij code stylesheet has the policy embedded. if it was intellij who inserted that line then fine